elytron/src/main/java/org/wildfly/extension/elytron/RealmParser.java

```diff
@@ -92,7 +92,30 @@
             .setMarshallDefaultValues(true)
             .build();
     private final PersistentResourceXMLDescription filesystemRealmParser_14_0 = builder(PathElement.pathElement(ElytronDescriptionConstants.FILESYSTEM_REALM), null)
-            .addAttributes(FileSystemRealmDefinition.ATTRIBUTES)
+            .addAttributes(FileSystemRealmDefinition.PATH)
+            .addAttribute(FileSystemRealmDefinition.RELATIVE_TO)
+            .addAttribute(FileSystemRealmDefinition.LEVELS)
+            .addAttribute(FileSystemRealmDefinition.ENCODED)
+            .addAttribute(FileSystemRealmDefinition.HASH_ENCODING)
+            .addAttribute(FileSystemRealmDefinition.HASH_CHARSET)
+            .build();
+    private final PersistentResourceXMLDescription filesystemRealmParser_15_0 = builder(PathElement.pathElement(ElytronDescriptionConstants.FILESYSTEM_REALM), null)
+            .addAttributes(FileSystemRealmDefinition.PATH)
+            .addAttribute(FileSystemRealmDefinition.RELATIVE_TO)
+            .addAttribute(FileSystemRealmDefinition.LEVELS)
+            .addAttribute(FileSystemRealmDefinition.ENCODED)
+            .addAttribute(FileSystemRealmDefinition.HASH_ENCODING)
+            .addAttribute(FileSystemRealmDefinition.HASH_CHARSET)
+            .build();
+    private final PersistentResourceXMLDescription filesystemRealmParser_15_1 = builder(PathElement.pathElement(ElytronDescriptionConstants.FILESYSTEM_REALM), null)
+            .addAttributes(FileSystemRealmDefinition.PATH)
+            .addAttributes(FileSystemRealmDefinition.RELATIVE_TO)
+            .addAttributes(FileSystemRealmDefinition.LEVELS)
+            .addAttributes(FileSystemRealmDefinition.ENCODED)
+            .addAttributes(FileSystemRealmDefinition.HASH_ENCODING)
+            .addAttributes(FileSystemRealmDefinition.HASH_CHARSET)
+            .addAttributes(FileSystemRealmDefinition.CREDENTIAL_STORE)
```

Comment:
It would be good to add a comment like this to indicate that this is a new attribute in 15_!:

```
.addAttributes(FileSystemRealmDefinition.CREDENTIAL_STORE) // new 
```

Reply:
Added